lib/extensions.js

```diff
@@ -63,6 +64,8 @@ function getBinaryName() {
     binaryName = flags['--sass-binary-name'];
   } else if (process.env.SASS_BINARY_NAME) {
     binaryName = process.env.SASS_BINARY_NAME;
+  } else if (process.env.npm_config_sass_binary_name) {
+    binaryName = process.env.npm_config_sass_binary_name;
```

Comment:
I guess that this is a bit the problem with the NPM documentation. It is not clearly written in the config part [1] that it works bidirectional. In fact NPM exposes **all** parameters (configuration, metadata...) to the env and is mentioned quiet hidden in the documentation like [2] and [3]. This is actually a quiet handy mechanism for configuration and also used by other projects [4].

It might make sense enhance this in the upstream npm documentation.

[1] https://docs.npmjs.com/misc/config#environment-variables
[2] https://docs.npmjs.com/misc/scripts#current-lifecycle-event
[3] https://docs.npmjs.com/files/package.json#config
[4] https://github.com/Medium/phantomjs/blob/master/install.js#L23